### PR TITLE
Add documentation for Clickatell integration

### PR DESCRIPTION
**Pull request by fadkeabhi**
Add documentation for Clickatell integration with Novu

### CODE REVIEW

File: channels-and-providers/sms/clickatell.mdx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: 'Clickatell'
+description: 'Learn how to use the Clickatell provider to send sms notifications using Novu'
+---
+
+You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform with a single API to create multi-channel experiences.
```
Comment (sumitsaurabh927) [inline channels-and-providers/sms/clickatell.mdx:6]: I don't understand the '...with single API to create multi-channel experiences' part. Is it about Novu?

If yes, I recommend removing it entirely. If someone is on this page, they already know what Novu can do and are interested in getting it to work with Clickatell so the focus should be on that.

File: channels-and-providers/sms/clickatell.mdx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+# Getting Started
+
+To use the Clickatell provider in the SMS channel, the first step is to create a Clickatell account and add your API key to the Clickatell integration on the Novu platform.
```
Comment (sumitsaurabh927) [inline channels-and-providers/sms/clickatell.mdx:10]: 'To use Clickatell, you need to [create a Clickatell account](insert a link to the sign-up page of Clickatell) and get your API key. You'll need to plug this into Clickatell integration on [Novu web](insert link to the integration section of Novu web)

File: images/channels-and-providers-sms-clickatell-3.png
Comment (sumitsaurabh927) [inline images/channels-and-providers-sms-clickatell-3.png:1]: For these images, can you use red as the colour to highlight the steps? You've used red in rectangles above and it'll be good to have a uniform colour. 🙂

File: images/channels-and-providers-sms-clickatell-4.png
Comment (sumitsaurabh927) [inline images/channels-and-providers-sms-clickatell-4.png:1]: Use red here as well!

**Review by sumitsaurabh927 — CHANGES_REQUESTED** [review]

**Comment by sumitsaurabh927** [comment]
> Added documentation for Clickatell integration with Novu

Great work @fadkeabhi 

Just for future reference, we write commit messages in imperative sense, so instead of `Added documentation for Clickatell integration`, it'll be `Add  documentation for Clickatell integration`. You can check more [here](https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53?permalink_comment_id=3497571#example-for-a-commit-message)

Thanks for contributing! 🎉

**Comment by fadkeabhi** [comment]
@sumitsaurabh927 I have applied requested changes. Can you take a look at it again.

File: channels-and-providers/sms/clickatell.mdx [hunk 3]
```diff
@@ -3,7 +3,7 @@ title: 'Clickatell'
 description: 'Learn how to use the Clickatell provider to send sms notifications using Novu'
 ---
 
-You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform with a single API to create multi-channel experiences.
+You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform.
```
Comment (sumitsaurabh927) [inline channels-and-providers/sms/clickatell.mdx:6]: You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS messages to your customers using the Novu Platform.

Let's see how to do it:

File: channels-and-providers/sms/clickatell.mdx [hunk 4]
```diff
@@ -7,7 +7,7 @@ You can use the [Clickatell](https://www.clickatell.com/) provider to send SMS
 
 # Getting Started
 
-To use the Clickatell provider in the SMS channel, the first step is to create a Clickatell account and add your API key to the Clickatell integration on the Novu platform.
+To use the [Clickatell](https://www.clickatell.com/) provider in the SMS channel, the first step is to [create a Clickatell account](https://www.clickatell.com/sign-up/) and add your API key to the Clickatell integration on the [Novu](https://web.novu.co/integrations) platform.
```
Comment (sumitsaurabh927) [inline channels-and-providers/sms/clickatell.mdx:10]: '...your API key to the Clickatell integration on the [Novu](https://web.novu.co/integrations) platform.' -> '...on the Novu web dashboard'

**Comment by sumitsaurabh927** [comment]
> @sumitsaurabh927 I have applied requested changes. Can you take a look at it again.

Dropped a couple of comments. Please address!

**Comment by fadkeabhi** [comment]
@sumitsaurabh927 I have applied requested changes. Can you take a look at it again.

File: channels-and-providers/sms/clickatell.mdx (hunk 3 above)
Comment (sumitsaurabh927) [reply channels-and-providers/sms/clickatell.mdx:6]: @fadkeabhi you need to update this as well!

Comment (fadkeabhi) [reply channels-and-providers/sms/clickatell.mdx:6]: @sumitsaurabh927 I have changed it already. Please take a look at updated file.

Comment (sumitsaurabh927) [reply channels-and-providers/sms/clickatell.mdx:6]: Oh, my bad!

**Review by sumitsaurabh927 — APPROVED** [review]

**Pull request merged**